tests/integration/test_consultation_page.py

```diff
@@ -1,20 +1,23 @@
 import pytest
 
-from consultation_analyser.factories import ConsultationWithAnswersFactory, UserFactory
+from consultation_analyser.consultations.dummy_data import create_dummy_consultation_from_yaml
+from consultation_analyser.consultations.models import Question
+from consultation_analyser.factories import UserFactory
 from tests.helpers import sign_in
 
 
 @pytest.mark.django_db
 def test_consultation_page(django_app):
     user = UserFactory()
-    consultation = ConsultationWithAnswersFactory(users=(user))
+    consultation = create_dummy_consultation_from_yaml()
+    consultation.users.add(user)
 
     sign_in(django_app, user.email)
 
     consultation_slug = consultation.slug
-    question = consultation.section_set.first().question_set.first()
+    question = Question.objects.filter(consultation=consultation).first()
     homepage = django_app.get(f"/consultations/{consultation_slug}/")
```

Comment:
A bit picky, so I apologise, but is `homepage` the right variable name here?

Reply:
probably not - consultations or similar I think

Reply:
Now updated https://github.com/i-dot-ai/consult/blob/77148101c0e62e4d5cdf29d2cbf8b0401aa2c830/tests/integration/test_consultation_page.py#L19.